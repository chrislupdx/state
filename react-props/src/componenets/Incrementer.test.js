import React from 'react';
import { shallow } from 'enzyme';
import Incrementer from './Incrementer';

describe('incrementer component', () => {
  it('renderes an incrementer', () => {
    const wrapper = shallow(<Incrementer />);
    expect(wrapper).toMatchSnapshot();
  });

    
});
