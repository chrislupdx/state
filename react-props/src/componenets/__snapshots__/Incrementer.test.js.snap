// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`incrementer component renderes an incrementer 1`] = `
<Fragment>
  <p>
    current count: 
    0
  </p>
  <button
    onClick={[Function]}
  >
    increment
  </button>
</Fragment>
`;
